format/hint/schema.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": "string"
+    },
+    "value": {
+      "type": ["string","null"]
```

Comment:
`"null"` is not a primitive json schema type (http://json-schema.org/latest/json-schema-core.html#anchor8). `null` is, but:

1. if the goal is to make the property optional, removing it from `required` is probably cleaner
2. having a null value here doesn't make sense for me, as an empty hint is basically unusable